pkg/session/bootstrap.go

```diff
@@ -609,7 +609,7 @@ const (
 
 	// CreateDistFrameworkMeta create a system table that distributed task framework use to store meta information
 	CreateDistFrameworkMeta = `CREATE TABLE IF NOT EXISTS mysql.dist_framework_meta (
-        host VARCHAR(100) NOT NULL PRIMARY KEY,
+        host VARCHAR(261) NOT NULL PRIMARY KEY,
```

Comment:
there is a `dispatcher_id` in global_task table, maybe change it too, in case we use it later